lib/munster/jobs/processing_job.rb

```diff
@@ -5,14 +5,23 @@
 module Munster
   class ProcessingJob < ActiveJob::Base
     def perform(webhook)
+      webhook.class.transaction do
```

Comment:
**issue:** #with_lock opens transaction internally. Is there a need to open another transaction?

See rails code:
https://github.com/rails/rails/blob/36c1591bcb5e0ee3084759c7f42a706fe5bb7ca7/activerecord/lib/active_record/locking/pessimistic.rb#L92

Reply:
None - I didn't know it does! thanks!